awx/main/management/commands/run_wsrelay.py

```diff
@@ -121,13 +122,14 @@ def handle(self, *arg, **options):
             return
 
         try:
-            my_hostname = Instance.objects.my_hostname()
+            my_hostname = Instance.objects.my_hostname()  # This relys on settings.CLUSTER_HOST_ID which requires database connection
```

Comment:
```suggestion
            my_hostname = Instance.objects.my_hostname()  # This relies on settings.CLUSTER_HOST_ID which requires database connection
```

Reply:
Do we know that we will always have a database connection here, or do we need to deal with a possible exception here?

Reply:
if db connection is down we crash (correct behavior) and rely on supervisor to restart wsrelay